app/pb_kits/playbook/pb_currency/_currency.scss

```diff
@@ -26,7 +26,7 @@
   [class^=pb_currency_wrapper] {
     display: flex;
 
-    & > [class*=dollar_sign] {
+    & > [class*=currency_indicator] {
```

Comment:
At first inspection, it doesn't look like this classname is getting included in the rails kit anymore.

Reply:
Good catch. Updated.